webservices/resources/elections.py

```diff
@@ -322,7 +322,7 @@ def filter_candidates(query, kwargs):
     query = query.filter(
         CandidateHistory.two_year_period <= kwargs['cycle'],
         CandidateHistory.two_year_period > (kwargs['cycle'] - duration),
-        CandidateHistory.election_years.any(kwargs['cycle']),
+        CandidateHistory.cycles.any(kwargs['cycle']),
```

Comment:
@PaulClark2 @jwchumley can FEC fix the null district candidates? If I remember correctly, there are all house candidates and there are only about 50 of them.j